lib/services/keyVault/lib/keyVaultClient.js

```diff
@@ -3666,6 +3700,16 @@ KeyVaultClient.prototype.getSecretVersions = function (vaultBaseUrl, secretName,
     if (maxresults !== null && maxresults !== undefined && typeof maxresults !== 'number') {
       throw new Error('maxresults must be of type number.');
     }
+    if (maxresults !== null && maxresults !== undefined) {
+      if (maxresults > 25)
```

Comment:
For now this is the limitation as stated by REST API and this issue: https://github.com/Azure/azure-sdk-for-node/issues/1842
I haven't seen any ask for change in this value.